.github/actions/image-scan/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Image Vulnerability Scan
+description: Run image vulnerability scans
+inputs:
+  image:
+    description: The docker image to scan
+    required: true
+runs:
+  using: "composite"
+  steps:
```

Comment:
If I understand correctly, these steps are currently set up to run synchronously, but they don't depend on each other so theoretically they could run in parallel as separate jobs. Is that correct?

Reply:
I think your example is describing what I'm saying – anchore-scan and trivy-scan need to be separate jobs in order to run in parallel. Putting them as separate steps in a single job or composite action will run them in serial. I don't know of a way to make the various steps in a `steps` section of a github action run in parallel.